rclcpp_components/cmake/rclcpp_components_register_nodes.cmake

```diff
@@ -13,38 +13,38 @@
 # limitations under the License.
 
 #
-# Register a node plugin with the ament resource index.
+# Register an rclcpp_component with the ament resource index.
```

Comment:
`rclcpp_component` -> `rclcpp component`

not using suggestion feature because of DCO :(